pyaedt/icepak.py

```diff
@@ -3834,11 +3844,11 @@ def assign_solid_block(
         """
         if not self.modeler.get_object_from_name(object_name).solve_inside:
             self.logger.add_error_message(
-                "Please use ``assign_hollow_block`` function with this object as" "``solve_inside`` is ``False``."
+                "Use ``assign_hollow_block`` function with this object as" "``solve_inside`` is ``False``."
```

Comment:
```suggestion
                "Use the ``assign_hollow_block()`` method with this object as ``"solve_inside`` is ``False``."
```
I know with line limits, articles like "the" have often been dropped in the past. However, omission does reduce readability, so we are encouaging more readable messages but asking developers to add them.